src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -21,9 +21,10 @@
           </slot>
         </span>
         <span class="k-multiselect-selected-icon-container">
-          <KTooltip
-            v-if="item.selected && item.disabled"
-            :label="item.disabledTooltipText"
+          <component
+            :is="item.disabledTooltipText ? 'KTooltip' : 'div'"
```

Comment:
issue: you cannot render a `div` inside of a `span`

Can this be another `span` instead?

Reply:
My bad, fixed